### ISSUE TITLE
Informative References (comment 20)

### ISSUE THREAD

**Issue by LPardue** [quicwg/ops-drafts#544]
Informative References: The target URL for [IPIM] leads to a page titled "Principles for Measurability in Protocol Design". Is this the correct link for the reference that is listed in the document? Or does the title of this reference need to be updated? 

Original:

       [IPIM]     Allman, M., Beverly, R., and B. Trammell, "In-Protocol
                  Internet Measurement (arXiv preprint 1612.02902)", 9
                  December 2016, <https://arxiv.org/abs/1612.02902>.

**Comment by mirjak** [comment]
I think this probably should be updated but @britram knows best!

**Comment by britram** [comment]
Yep, the proper title is "Principles for Measurability in Protocol Design"
